pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -3782,6 +3784,99 @@ func (r *ClickHouseReader) GetMetricAttributeValues(ctx context.Context, req *v3
 	return &attributeValues, nil
 }
 
+func (r *ClickHouseReader) GetLogAttributeKeys(ctx context.Context, req *v3.FilterAttributeKeyRequest) (*v3.FilterAttributeKeyResponse, error) {
+	var query string
+	var err error
+	var rows driver.Rows
+	var response v3.FilterAttributeKeyResponse
+
+	if len(req.SearchText) != 0 {
+		query = fmt.Sprintf("select distinct tagKey, tagDataType from  %s.%s where tagKey ILIKE $1 and tagType=$2 limit $3", r.logsDB, r.logsTagAttributeTable)
+		rows, err = r.db.Query(ctx, query, fmt.Sprintf("%%%s%%", req.SearchText), req.TagType, req.Limit)
```

Comment:
The `tagType` from `FilterAttributeKeyRequest` is not the tag type of the search text. It is the tag type of the aggregate attribute, which doesn't necessarily need to be the same as the filter key. Here the client is sending the aggregate operator, attribute, and it's tag type as a prior user selection context; it wouldn't know the tag type of the key before hand?

Reply:
Got it, removing this so that frontend can get the tags without the tagType and it can be used for aggregate attribute once I add the support for that.